ledger/accountdb.go

```diff
@@ -5383,3 +5399,51 @@ func (qs *stateProofVerificationDbQueries) lookupData(stateProofLastAttestedRoun
 	err := db.Retry(queryFunc)
 	return &data, err
 }
+
+// stateProofVerificationTable returns all verification data currently committed to the given table.
+func stateProofVerificationTable(ctx context.Context, tx *sql.Tx, tableName string) ([]ledgercore.StateProofVerificationData, error) {
+	var result []ledgercore.StateProofVerificationData
+	queryFunc := func() error {
+		selectQuery := fmt.Sprintf("SELECT verificationdata FROM %s ORDER BY targetstateproofround", tableName)
+		rows, err := tx.QueryContext(ctx, selectQuery)
+
+		if err != nil {
+			return err
+		}
+
+		defer rows.Close()
+
+		// Clear `res` in case this function is repeated.
+		result = result[:0]
+		for rows.Next() {
+			var rawData []byte
+			err = rows.Scan(&rawData)
+			if err != nil {
+				return err
+			}
+
+			var record ledgercore.StateProofVerificationData
+			err = protocol.Decode(rawData, &record)
+			if err != nil {
+				return err
+			}
+
+			result = append(result, record)
+		}
+
+		return nil
+	}
+
+	err := db.Retry(queryFunc)
+	return result, err
```

Comment:
CatchpointStateProofVerification is used in the catchpointdump utility, and I think it would be weird to specifically wrap the StateProofVerification function's result with catchpointStateProofVerificationData